scribble-lib/scribble/manual-racket.js

```diff
@@ -15,14 +15,130 @@ AddOnLoad(function() {
     }
 })
 
+// cache of source urls
+var cache = new Map();
+
+function ParseSource(source, mod_path, single_collection) {
+    let source_re = /^(https|git):\/\/\github.com\/([^\/]+)\/([^\?\/]+?)(\/|)(\.git|)(\?path=(.+)|)(#[A-Za-z0-9_-]*|)$/;
```

Comment:
`let` doesn't work on old browsers. If you care about backward compatibility, you might want to use `var` instead.